src/SlashCommands.js

```diff
@@ -282,6 +282,63 @@ var commands = {
             }
         }
         return reject(this.getUsage());
+    }),
+
+    // Verify a user, device, and pubkey tuple
+    verify: new Command("verify", "<userId> <deviceId> <deviceSigningKey>", function(room_id, args) {
+        if (args) {
+            var matches = args.match(/^(\S+) +(\S+) +(\S+)?$/);
```

Comment:
Well, I guess I was trying to avoid having a complex regex considering all the information is checked against APIs which accept arbitrary strings.  Is there any reason to make this more specific? Any recommendations?